services/src/test/java/org/fao/geonet/api/records/MetadataAssociatedApiTest.java

```diff
@@ -123,6 +123,7 @@ public void getRelatedNonExistent() throws Exception {
     }
 
     @Test
+    @Ignore("indexation troubles, multilingual and iso19115-3.2018 ?")
```

Comment:
This https://github.com/fxprunayre/core-geonetwork/commit/c3215cc7ac5c59c1f414f82019be02196fc4319c could help here Christophe. Can you cherry-pick?